meteor-app/imports/api/orders/schema.test.js

```diff
@@ -167,7 +165,6 @@ describe('schema', () => {
       it('Return database query', () => {
         expect(Orders.find().fetch()[0].totalPrice).to.equal(9900)
```

Comment:
If you use `.findOne()` it does the same thing as `.find().fetch()`
eg `expect(Orders.findOne().totalPrice).to.equal(9900)`

BUT

1) You are assuming that if you fetch all the records that the first one returned is the first one you inserted, and you can't assume that. There is no guarantee of order
2) This test is run after every good order is inserted, but it's only checking the first one each time.
3) You are checking against the value 9900 - if you change the sample data and forget this, the test will break

Your code would be much better if you save the `_id` of the inserted record into a variable, and then use that `_id` to find the record, and find the record using that id. eg

```
const orderId = Orders.insert(good)
const order = Orders.findOne(orderId)
// compare order.totalPrice to good.totalPrice
```

An extension to this would be to do more checking,
```
let ok = true
if (good. orderedParts && good. orderedParts.length) {
  good. orderedParts.forEach((part,i) => {
    Object.keys(part).forEach(key => {
      if (part[key] !== good.orderedParts[i][key]) {
        ok = false
      }
    })
  })
}
// At the end of this `ok` will be false if something didn't match
```